proto/zetachain/zetacore/pkg/chains/chains.proto

```diff
@@ -88,6 +88,19 @@ enum Consensus {
   op_stack = 3;
 }
 
+// CCTXGateway describes for the chain the gateway used to handle CCTX outbounds
+enum CCTXGateway {
+  option (gogoproto.goproto_enum_stringer) = true;
+
+  // zevm is the internal CCTX gateway to process outbound on the ZEVM in
+  // ZetaChain only used for ZetaChain chains
```

Comment:
The comment seems inconsistent with the one for observers as they do not talk about the inbound; if we want to treat this as an outbound-only gateway, that should be mentioned in the comments or the naming.

Reply:
For the initial implementation, I think it is simpler to just have a `ValidateInbound` method for crosschain keeper that can be called from anywhere. In the future, I think this would make more sense to only allow calling `ValidateInbound` from a CCTXGateway (like passing a `SetOnInboundValidation(func ...)` method as part of the interface) this would make the design a bit more complex but more intuitive. This is why I wanted to keep the name more general `CCTXGateway` instead of `OutboundGateway`.
But since renaming the interface would be trivial, maybe we can call it `OutboundGateway` for now if this makes clearer

Reply:
Okay, that makes sense; maybe adding this to the comments might be enough.

Reply:
[fc6f66b](https://github.com/zeta-chain/node/pull/2279/commits/fc6f66b8b0c9c283ee9e5fe08c57af54e6aebbe3)